File: twakkal_lwp/src/main/java/com/yassin/sfax/tawakkolalaallah/MyPreferencesActivity.java
package com.yassin.sfax.tawakkolalaallah;

import com.yassin.sfax.tawakkolalaallah.R;

import android.os.Bundle;
import android.preference.PreferenceActivity;

public class MyPreferencesActivity extends PreferenceActivity {
  @SuppressWarnings("deprecation")
@Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    addPreferencesFromResource(R.xml.livewallpaper_settings);

  }



} 

File: twakkal_lwp/src/main/java/com/yassin/sfax/tawakkolalaallah/MyPreferencesActivity.kt
package com.yassin.sfax.tawakkolalaallah

import android.os.Bundle
import android.preference.PreferenceActivity

class MyPreferencesActivity : PreferenceActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        addPreferencesFromResource(R.xml.settings)
    }
}